assets/html/Project.toml

```diff
@@ -1,3 +1,7 @@
 [deps]
 Documenter = "e30172f5-a6a5-5a46-863b-614d45cd2de4"
 DocumenterTools = "35a29f4d-8980-5a13-9543-d66fff28ecb8"
+libsass_jll = "47bcb7c8-5119-555a-9eeb-0afcc36cd728"
+
+[compat]
+libsass_jll = "~3.6"
```

Comment:
I probably had another version of this locally when I built the themes then? Or why do you think they changed?

Reply:
I didn't investigate it too carefully, but if it had been a libsass difference, then I would expect the `documenter-*` CSS files to be outdated as well.